### DISCUSSION TITLE
Can you make current directory as the Excalidraw folder, if so, how?

### DISCUSSION THREAD

**Discussion by TheFern2** [zsviczian/obsidian-excalidraw-plugin#1839] [Q&A] [answered]
Basically the title

**Comment by Sandr-aa** [comment, marked as answer]
You have to manually change the path in the settings under Basic, the files need to be moved into the new folder manually.
![2024-08-08_19-12](https://github.com/user-attachments/assets/3aeafc6b-3953-45ba-96e1-c9d716115314)
